health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CampaignType.js

```
@@ -1,7 +1,9 @@
 import React, { useState, useMemo, useRef, useEffect, Fragment } from "react";
+import { UploadIcon, FileIcon, DeleteIconv2, Toast, Header } from "@egovernments/digit-ui-react-components";
 import { useTranslation } from "react-i18next";
-import { AlertCard, LabelFieldPair } from "@egovernments/digit-ui-components";
-import { Button, CardText, Dropdown, ErrorMessage, PopUp, Stepper, TextBlock , Card , HeaderComponent} from "@egovernments/digit-ui-components";
+import { LabelFieldPair } from "@egovernments/digit-ui-components";
+import { AlertCard, LabelFieldPair, Button, CardText, Dropdown, ErrorMessage, PopUp, Stepper,HeaderComponent, TextBlock, Card } from "@egovernments/digit-ui-components";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'LabelFieldPair'. Consider to delete it or rename it.

'LabelFieldPair' is defined here:


(lint/suspicious/noRedeclare)